index.md

```diff
@@ -10,7 +10,7 @@ description: OceanHackWeek home
 :class: important
 
 **Applications for OceanHackWeek 2023 are now open!**  
-OceanHackWeek 2023 will be a hybrid event with a global virtual event; a smaller in-person gathering at the University of Washington in Seattle; and a small Australian gathering.
+OceanHackWeek 2023 will be a hybrid event with a global virtual component; a 20-25 in-person component at the University of Washington in Seattle, WA, USA; and a small in-person component in Australia (likely Dunsborough).
```

Comment:
I personally like/prefer the pattern you used of describing OHW as a whole as an "event" with "components". But that's not how we describe it elsewhere in the website and the application, where we describe each of the 3 components as an "event". So, I edited your text to be consistent. If you agree, go ahead and accept my commit, then feel free to merge the PR. Thanks!
```suggestion
OceanHackWeek 2023 will be a hybrid workshop with a global virtual event; a 20-25 in-person event at the University of Washington in Seattle, WA, USA; and a small in-person event in Australia (likely Dunsborough).
```